tests/unit/test_utils.py

```diff
@@ -16,6 +16,17 @@ class TestPytestMrackUtils:
     def test_get_shortname(self, hostname, expected):
         assert get_shortname(hostname) == expected
 
+    @pytest.mark.parametrize(
+        "hostname,domain,expected",
+        [
+            ("abc.test", "bar", "abc.test"),
```

Comment:
A host under domain `bar` cannot have a FQDN not containing the domain `bar`. In this case the the `abc.test` must be either under TLD domain `test` or the FQDN should be e.g. `abc.test.bar`

Reply:
Point of this test is that when name is fqdn, domain is ignored/overridden by whatever is in the hostname.
**bar** is there only to make it clear.

Reply:
Resolved.